modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractCSharpCodegen.java

```diff
@@ -471,7 +471,7 @@ public void postProcessModelProperty(CodegenModel model, CodegenProperty propert
         if (property.isEnum && !property.vendorExtensions.containsKey(this.zeroBasedEnumVendorExtension)) {
             if (Boolean.TRUE.equals(this.zeroBasedEnums)) {
                 property.vendorExtensions.put(this.zeroBasedEnumVendorExtension, true);
-            } else if (!Boolean.FALSE.equals(this.zeroBasedEnums)) {
+            } else {
```

Comment:
I don't think we want this change.

Reply:
undo via https://github.com/OpenAPITools/openapi-generator/commit/7df05a53b53bf605b52eadc37af4535acf5e8a70